flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/test/java/org/apache/flink/cdc/connectors/mysql/source/NewlyAddedTableITCase.java

```diff
@@ -214,29 +214,6 @@ public void testNewlyAddedTableForExistsPipelineThriceWithAheadBinlog() throws E
                 "address_shenzhen");
     }
 
-    @Test
-    public void testNewlyAddedTableForExistsPipelineSingleParallelism() throws Exception {
-        testNewlyAddedTableOneByOne(
-                1,
-                FailoverType.NONE,
-                FailoverPhase.NEVER,
-                false,
-                "address_hangzhou",
-                "address_beijing");
-    }
-
-    @Test
-    public void testNewlyAddedTableForExistsPipelineSingleParallelismWithAheadBinlog()
-            throws Exception {
-        testNewlyAddedTableOneByOne(
-                1,
-                FailoverType.NONE,
-                FailoverPhase.NEVER,
-                true,
-                "address_hangzhou",
-                "address_beijing");
-    }
-
```

Comment:
@Karl-WangSK Thanks for your contribution, but they are different cases, why we need to remove the two cases?